lib/qs.rb

```diff
@@ -48,6 +61,9 @@ def self.redis_config
   class Config
     include NsOptions::Proxy
 
+    option :serializer,   Proc, :default => proc{ |p| ::JSON.dump(p) }
+    option :deserializer, Proc, :default => proc{ |p| ::JSON.load(p) }
```

Comment:
@jcredding should we put a `require 'json'` statement insight these procs?  I don't like a user having to remember to manually require it if they haven't overridden the defaults.  What do you think?

Also, what is `p` here?  Should we use more explicit var names?